ui/src/literature/components/__tests__/__snapshots__/LiteratureItem.test.jsx.snap

```diff
@@ -302,6 +308,9 @@ exports[`LiteratureItem renders with all props set, including sub props 1`] = `
       }
       wrapperClassName="di"
     />
+    <ConferenceInfoList
```

Comment:
` renders with all props set, including sub props 1` should have minimal `conference_info` too.